tests/org.eclipse.n4js.ide.tests/src/org/eclipse/n4js/ide/tests/builder/IncrementalBuilderFileChangesTest.xtend

```diff
@@ -108,4 +109,95 @@ class IncrementalBuilderFileChangesTest extends AbstractIncrementalBuilderTest {
 		assertNull("index should no longer contain an entry for A.n4js", getResourceDescriptionFromIndex(fileA));
 		outputFileA.assertNotExists();
 	}
+
+	@Test
+	def void testFixFileBetweenSessions() {
```

Comment:
meaning of "sessions" wasn't immediately obvious to me. Maybe "...ServerSessions" or "...ServerExecutions"?